gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/tasks/snapshots/Register.kt

```diff
@@ -252,10 +232,6 @@ fun getSnapshotUploadTaskName(variantName: String): String {
   return "${EMERGE_TASK_PREFIX}UploadSnapshotBundle${variantName.capitalize()}"
 }
 
-private val Project.firstPartySnapshotsEnabled
```

Comment:
alternatively, we could print out a warning telling users to remove this property since it is unused but i don’t think anyone was using it.